cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Babbage/LeadershipSchedule.hs

```diff
@@ -87,33 +86,22 @@ hprop_leadershipSchedule = H.integrationRetryWorkspace 2 "babbage-leadership-sch
   work <- H.note tempAbsPath'
 
   ----------------Need to register an SPO------------------
-
- -- We get our UTxOs from here
-  utxoVKeyFile <- H.note $ work </> "utxo-keys/utxo1.vkey"
-  utxoSKeyFile <- H.note $ work </> "utxo-keys/utxo1.skey"
-
-  utxoAddr <- execCli
-                [ "address", "build"
-                , "--testnet-magic", show @Int testnetMagic
-                , "--payment-verification-key-file", utxoVKeyFile
-                ]
-
-  void $ execCli' execConfig
-      [ "query", "utxo"
-      , "--address", utxoAddr
-      , "--cardano-mode"
-      , "--testnet-magic", show @Int testnetMagic
-      , "--out-file", work </> "utxo-1.json"
-      ]
-
-  H.cat $ work </> "utxo-1.json"
+  let utxoAddr = Text.unpack $ paymentKeyInfoAddr $ head wallets
+      utxoSKeyFile = paymentSKey . paymentKeyInfoPair $ head wallets
+  void $ H.execCli' execConfig
+    [ "conway", "query", "utxo"
+    , "--address", utxoAddr
+    , "--cardano-mode"
+    , "--testnet-magic", show @Int testnetMagic
+    , "--out-file", work </> "utxo-1.json"
+    ]
 
   utxo1Json <- H.leftFailM . H.readJsonFile $ work </> "utxo-1.json"
-  UTxO utxo1 <- H.noteShowM $ decodeEraUTxO sbe utxo1Json
-  txin <- H.noteShow =<< H.headM (Map.keys utxo1)
+  UTxO utxo1 <- H.noteShowM $ H.noteShowM $ decodeEraUTxO sbe utxo1Json
```

Comment:
why duplicated `noteShowM`?

Reply:
copy paste error, I'll fix it once the branch passes CI